al2.pkr.hcl

```diff
@@ -174,12 +176,23 @@ build {
 
   provisioner "shell" {
     environment_vars = ["AMI_TYPE=${source.name}"]
-    script           = "scripts/enable-ecs-agent-inferentia-support.sh"
+    script           = "scripts/al2/install-kernel5dot10.sh"
+  }
+
+  ### reboot worker instance to install kernel update. enable-ecs-agent-inferentia-support needs
+  ### new kernel (if there is) to be installed.
+  provisioner "shell" {
```

Comment:
Q. I saw we reboot the worker instance for AL2023 in [al2023.pkr.hcl#L144-L152](https://github.com/aws/amazon-ecs-ami/blob/f09d7d5fe5d2d3c215da7f5d377dc2c062e6a3ad/al2023.pkr.hcl#L144-L152), but should we only reboot it when the kernel has to be updated?

Reply:
Thanks!!!